src/lib/passwordGenerator.ts

```diff
@@ -0,0 +1,67 @@
+interface passwordProps {
+  includeUpper : boolean;
+  includeLower : boolean;
+  includeSymbol : boolean;
+  includeNumber : boolean;
+}
+
+export default class PasswordGenerator {
+  characters : string = "";
+  passwordLen : number = 12;
```

Comment:
Might be good to include this as an input prop, even if we have a fixed generated password length for the app.

Reply:
There is an input prop for initiating the class. The parameter is called pwdLength and it's an optional. I set the default since we're not accepting user input to determine the length.